edumanage/context_processors.py

```diff
@@ -1,4 +1,6 @@
 from django.conf import settings
+from django.core.urlresolvers import reverse
+import edumanage
```

Comment:
A fully qualified import is probably preferable here, `edumanage.views.set_language` is used below and is probably working due to to it being imported elsewhere. Better to be explicit.

Reply:
Thanks, just pushed in a commit changing this to fully qualified import.